package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "skynet-js",
-  "version": "4.0.27-beta",
```

Comment:
This is not how semver works, afaik. We need to release 4.0.28 and mark it non-beta.

If I was a user of this library and I couldn't look up release dates, I'd be confident to say that the order of versions is 3.0.2, 4.0.0, 4.0.1-beta, ...

Reply:
yeah it's total true, 4.1.0 is the way to go I guess